client/app/account/settings/settings.html

```diff
@@ -1,14 +1,43 @@
 <div class="container">
   <div class="row">
     <div class="col-sm-12">
-      <h1>Change Password</h1>
+      <h1>Account Settings</h1>
     </div>
     <div class="col-sm-12">
       <form class="form" name="form" ng-submit="vm.changePassword(form)" novalidate>
 
         <div class="form-group">
-          <label>Current Password</label>
+          <label>Username</label>
+          <!-- current username here -->
+          <div class="editContainer">
+            <p ng-show='true'> John Doe{{ $ctrl.getCurrentUser().name }}</p>
+            <input type="username" placeholder="" class="form-control" ng-hide='true'/>
+            <button class="btn btn-lg btn-primary editBtn" type="edit" ng-show=true>
+              Edit
+            </button>
+            <button class="btn btn-lg btn-primary editBtn" type="Save" ng-hide=true>
```

Comment:
Set ng-hide to a variable in the controller. Have ng-click on the edit button change the variable between true/false.